rs/index/src/collection/collection.rs

```diff
@@ -190,6 +208,8 @@ impl<Q: Quantizer + Clone + Send + Sync + 'static> Collection<Q> {
                     continue;
                 }
 
+                INTERNAL_METRICS.num_searchable_docs_set(&collection_name, 0);
```

Comment:
I don't think you need to set this (also this value should not be 0, since this is a normal collection that has not been read). I think you can just remove this line.